tests/vmcheck/overlay.sh

```diff
@@ -13,26 +13,9 @@ if test -z "${INSIDE_VM:-}"; then
 
     vm_rsync
 
-    # ✀✀✀ BEGIN selinux-policy hack (part 1) for
-    # https://github.com/fedora-selinux/selinux-policy-contrib/pull/45
```

Comment:
What about centos though?

Reply:
This hack only really applied to Fedora. It always worked on CentOS.